### ISSUE TITLE
Where are the training labels?

### ISSUE THREAD

**Issue by akssieg** [sanatladkat/floor-plan-object-detection#1]
I downloaded the dataset but how to compare the trained model with the labels? The labels aren't available on the roboflow.

**Comment by iqbbal** [comment]
hello @sanatladkat, can you provide pre-trained and also dataset? thanks!

**Comment by sanatladkat** [comment]
Hi @akssieg @iqbbal,,

Thank you for your questions!

### Regarding the labels:
The training labels and dataset that I used for training the model cannot be disclosed, unfortunately. However, I have shared the `best.pt` model file, which should be useful for your application.

### Comparing the trained model with labels:
Since the dataset and labels cannot be shared, you would need to train the model on your own data. You can use the following dataset to train your model:

**Training Dataset:**  
[Floor Plan Multiple Dataset on Roboflow](https://universe.roboflow.com/walldetect-f9eio/floor_plan_multiple)

Once you train your own model, you can compare the predictions with the actual labels from your dataset.

### Pre-trained Model & Dataset:
While I cannot share the specific dataset or pre-trained model, you can train the model on the provided dataset or any other dataset that fits your use case.

Let me know if you need help with the training process or if you have any further questions!

Thanks!
